src/backend/agent.js

```diff
@@ -359,6 +363,11 @@ export default class Agent extends EventEmitter {
     window.removeEventListener('mousedown', this._onMouseDown, true);
     window.removeEventListener('mouseup', this._onMouseUp, true);
     window.removeEventListener('mouseover', this._onMouseOver, true);
+
+    this._updateSelectedFiber();
```

Comment:
Is this right? (that we update selection after stop?)

Reply:
One last time since the interval is cancelled.

Reply:
I guess I was just thinking about the reasons `stopInspectingDOM` might be called:
1. The user clicked on a DOM node (in which case, it's already selected in the tree)
2. The user clicked on the the toggle in the DevTools

Seems like in either case, we wouldn't need to actually call `_updateSelectedFiber` anymore?

Reply:
Ohh.... I'm wrong about the mouse down bit now I guess.